server/models/module_spec_schema.yaml

```diff
@@ -47,12 +47,12 @@ properties:
         type: array
         description: "Specification for how to give a form to the user and give values to the fetch/render functions"
         items:
-            $ref: '#/definitions/ftypes/any'
+            $ref: '#/definitions/param_types/any'
 
 definitions:
-    # FTypes: Form-element types. Each of these is a React component.
-    ftypes:
-        # _lib: properties that are used in more than one ftype.
+    # param_types: Types that module author can specify, which in turn define both the UI and the storage format (DType)
```

Comment:
`param_types` is not an improvement. The confusing part of "FType" isn't the "F": it's the word, "type".

Because it's not a type. The thing we're describing here is the entire contents of the "parameters" field. One element of that description is the "type" property; but we want to communicate that each definition is for _all the properties together_.

This should be `param_field` ... or maybe `field`? `parameter`?

Reply:
All I can say is I found the "f" confusing. Working through all of this, I maintain that the "ftype" *is* the parameter type. It's what's specified in the YAML, it completely defines the dtype (and not the other way around), and it's what the user sees. So I think just calling it the "type" is both simple and accurate.

Reply:
I'm not debating which name is best. I'm requesting a more-consistent pull request.

Prior to this pull request, there are three names for the same Thing:

* module spec: "parameter"
* module schema: "ftype"
* python: "ParamField"
* javascript: "parameter"

Once we merge this pull request (as it is above), there will _still_ be three names for the same Thing:

* module spec: "parameter"
* module schema: "param_type"
* python: "ParamField"
* javascript: "parameter"

You correctly identified a problem in my original code: I named the same Thing three different names. If we're going to go through the trouble of renaming, let's fix that problem.